test/integration/acceptance/tx_acceptance_test.cpp

```diff
@@ -0,0 +1,394 @@
+          shared_model::interface::StatelessValidTxResponse>>(status.get()));
+};
+auto checkStatelessInvalid = [](auto &status) {
+  ASSERT_NO_THROW(
+      boost::get<shared_model::detail::PolymorphicWrapper<
+          shared_model::interface::StatelessFailedTxResponse>>(status.get()));
+};
+
+auto checkProposal = [](auto &proposal) {
+  ASSERT_EQ(proposal->transactions.size(), 1);
+
+};
+auto checkStatefulInvalid = [](auto &block) {
+  ASSERT_EQ(block->transactions.size(), 0);
+};
+auto checkStatefulValid = [](auto &block) {
+  ASSERT_EQ(block->transactions.size(), 1);
+};
+/**
+ * @given non existent user
+ * @when sending  transaction to the ledger
+ * @then receive STATELESS_VALIDATION_SUCCESS status
+ *       AND STATEFUL_VALIDATION_FAILED on that tx
+ */
+TEST(AcceptanceTest, NonExistentCreatorAccountId) {
+  auto tx = shared_model::proto::TransactionBuilder()
+                .txCounter(2)
+                .createdTime(iroha::time::now())
+                .creatorAccountId(kNonUser)
+                .addAssetQuantity(kAdmin, kAsset, "1.0")
+                .build()
+                .signAndAddSignature(kAdminKeypair);
+
+  integration_framework::IntegrationTestFramework(1)
+      .setInitialState(kAdminKeypair)
+      .sendTx(tx, checkStatelessValid)
+      .checkProposal(checkProposal)
+      .checkBlock(checkStatefulInvalid)
+      .done();
+}
+
+/**
+ * @given some user
+ * @when sending 2 transactions with the same txCounter to the ledger
+ * @then receive STATELESS_VALIDATION_SUCCESS status on both txs
+ *       AND STATELESS_VALIDATION_SUCCESS on first
+ *       AND STATEFUL_VALIDATION_FAILED on second
+ */
+TEST(AcceptanceTest, DISABLED_DublicatedTxCounter) {
+  auto tx1 = shared_model::proto::TransactionBuilder()
+                 .txCounter(2)
+                 .createdTime(iroha::time::now())
+                 .creatorAccountId(kAdmin)
+                 .addAssetQuantity(kAdmin, kAsset, "1.0")
+                 .build()
+                 .signAndAddSignature(kAdminKeypair);
+  auto tx2 = shared_model::proto::TransactionBuilder()
+                 .txCounter(2)
+                 .createdTime(iroha::time::now())
+                 .creatorAccountId(kAdmin)
+                 .addAssetQuantity(kAdmin, kAsset, "1.0")
+                 .build()
+                 .signAndAddSignature(kAdminKeypair);
+
+  integration_framework::IntegrationTestFramework(1)
+      .setInitialState(kAdminKeypair)
+      .sendTx(tx1, checkStatelessValid)
+      .skipProposal()
+      .checkBlock(checkStatefulValid)
+      .sendTx(tx2, checkStatelessValid)
+      .skipProposal()
+      .checkBlock(checkStatefulInvalid)
+      .done();
+}
+
+/**
+ * @given some user
+ * @when sending  transactions with the Maximum txCounter possible to the ledger
+ * @then receive STATELESS_VALIDATION_SUCCESS status
+ *       AND STATEFUL_VALIDATION_SUCCESS on that tx
+ */
+TEST(AcceptanceTest, DISABLED_MaxTxCounter) {
```

Comment:
Use `iroha::model::all_perm_group` in ITF permission setting and enable all of the tests